pulsar-client-cpp/python/pulsar/schema/definition.py

```diff
@@ -444,8 +444,8 @@ def validate_type(self, name, val):
 
         for x in val:
             if type(x) != self.array_type.python_type():
-                raise TypeError('Array field ' + name + ' items should all be of type '
-                                + self.array_type.python_type())
+                raise TypeError('Array field ' + name + ' items should all be of type ' +
+                                str(self.array_type.type()))
```

Comment:
```suggestion
                                self.array_type.type())
```

The `type()` method already returns a `str`, there is no need to cast it to `str`.

Reply:
> The `type()` method already returns a `str`, there is no need to cast it to `str`.

Thanks for your comment, I just removed unnecessary casting, please kindly review.